lib/semian/circuit_breaker.rb

```diff
@@ -51,14 +51,13 @@ def mark_failed(error)
 
     def mark_success
       return unless half_open?
-      @success_count += 1
+      push_time(@successes, @success_count_threshold)
```

Comment:
@camilo it's a sliding window of a limited size. This is normally 2-5 elements long. Please see the definition of `push_item`.